addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -965,7 +965,7 @@ export class OdooEditor extends EventTarget {
                 }
             }
         }
-        if (sideEffect) {
+        if (sideEffect && !this._fixLinkMutatedElements) {
```

Comment:
I meant to put `this._activateContenteditable()` in a condition, not to put evverything else in this condition. Maybe it's the correct approach, I don't know for sure, but I find it kind of fishy that we don't want to dispatch the revert just because it turns out that we were editing links ?